exporter/nodes.go

```diff
@@ -306,11 +324,46 @@ func fetchNodePartitionMetrics(nodes []NodeMetric) map[string]*PartitionMetric {
 			partition.IdleCpus += node.IdleCpus
 			partition.RealMemory += node.RealMemory
 			partition.Weight += node.Weight
+			
+			nodeGres := parseGres(node.Gres)
+			for name, count := range nodeGres {
+				partition.TotalGres[name] += count
+			}
+			nodeGresUsed := parseGres(node.GresUsed)
+			for name, count := range nodeGresUsed {
+				partition.AllocGres[name] += count
+			}
 		}
 	}
 	return partitions
 }
 
+func parseGres(gres string) map[string]float64 {
```

Comment:
Sorry, maybe I'm misinterpreting this code, but will this ever return a map with more than 1 entry. At least, given your fixture data, I believe the answer is no. If I'm incorrect, please show me a counter example string.  In the following string: `gpu:tesla:2(IDX:0,1)` don't we just need the `2` and everything preceding it? Shouldn't the following regex work?

```golang
re := regexp.MustCompile(`^(.+):(\d+)\(`)
```

That would give us:

```golang
matches := re.FindStringSubmatch("gpu:tesla:1(IDX:0)")
// matches[1] == "gpu:tesla"
// matches[2] == "1"
```